pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -341,11 +341,12 @@ stdenv.mkDerivation {
       echo "-no_uuid" >> $out/nix-support/libc-ldflags-before
     ''
 
-    + ''
+    + optionalString (!stdenv.targetPlatform.isMusl) ''
```

Comment:
What's the reason behind this? Just because Hydra is building with musl doesn't necessarily mean that the glob can't fail on other platforms or C libraries.

Reply:
You are correct here, this needs more investigation to determine if it fails only on musl or on other platforms as well.

Reply:
trying to remove this completely for now.